huaweicloud/services/acceptance/rds/resource_huaweicloud_rds_pg_database_test.go

```diff
@@ -0,0 +1,141 @@
+
+	getPgDatabaseResp, err := pagination.ListAllItems(
+		getPgDatabaseClient,
+		"page",
+		getPgDatabasePath,
+		&pagination.QueryOpts{MarkerField: ""})
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS PostgreSQL database: %s", err)
+	}
+
+	getPgDatabaseRespJson, err := json.Marshal(getPgDatabaseResp)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS PostgreSQL database: %s", err)
+	}
+	var getPgDatabaseRespBody interface{}
+	err = json.Unmarshal(getPgDatabaseRespJson, &getPgDatabaseRespBody)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving RDS PostgreSQL database: %s", err)
+	}
+
+	database := utils.PathSearch(fmt.Sprintf("databases[?name=='%s']|[0]", dbName), getPgDatabaseRespBody, nil)
+	if database != nil {
+		return database, nil
+	}
+
+	return nil, fmt.Errorf("error retrieving RDS PostgreSQL database by instanceID %s and database %s", instanceId,
+		dbName)
+}
+
+func TestAccPgDatabase_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_rds_pg_database.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getPgDatabaseResourceFunc,
+	)
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:          func() { acceptance.TestAccPreCheck(t) },
+		ProviderFactories: acceptance.TestAccProviderFactories,
+		CheckDestroy:      rc.CheckResourceDestroy(),
+		Steps: []resource.TestStep{
+			{
+				Config: testPgDatabase_basic(name, "test_description"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "instance_id",
+						"huaweicloud_rds_instance.test", "id"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "owner", "root"),
+					resource.TestCheckResourceAttr(rName, "character_set", "UTF8"),
+					resource.TestCheckResourceAttr(rName, "lc_collate", "en_US.UTF-8"),
+					resource.TestCheckResourceAttr(rName, "description", "test_description"),
+					resource.TestCheckResourceAttrSet(rName, "size"),
+				),
+			},
+			{
+				Config: testPgDatabase_basic(name, "test_description_update"),
+				Check: resource.ComposeTestCheckFunc(
+					rc.CheckResourceExists(),
+					resource.TestCheckResourceAttrPair(rName, "instance_id",
+						"huaweicloud_rds_instance.test", "id"),
+					resource.TestCheckResourceAttr(rName, "name", name),
+					resource.TestCheckResourceAttr(rName, "owner", "root"),
+					resource.TestCheckResourceAttr(rName, "character_set", "UTF8"),
+					resource.TestCheckResourceAttr(rName, "lc_collate", "en_US.UTF-8"),
+					resource.TestCheckResourceAttr(rName, "description", "test_description_update"),
+				),
+			},
+			{
+				ResourceName:            rName,
+				ImportState:             true,
+				ImportStateVerify:       true,
+				ImportStateVerifyIgnore: []string{"template", "lc_ctype", "is_revoke_public_privilege"},
```

Comment:
Add description for import operation when ignoring fields.

Reply:
done